pkgs/applications/emulators/craftos-pc/default.nix

```diff
@@ -59,6 +60,13 @@ stdenv.mkDerivation rec {
     cp -R ${craftos2-rom}/* $out/share/craftos
   '';
 
+  passthru.tests = {
+    eval-hello-world = callPackage ./test-eval-hello-world { };
+    eval-periphemu = callPackage ./test-eval-periphemu { };
+  };
+
+  doCheck = true;
```

Comment:
Are there tests in the package? doCheck does not influence passthru.tests

Reply:
Whoops, my bad.